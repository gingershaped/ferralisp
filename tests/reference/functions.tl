(d #1 42)
(d F
    (q (
    (#1)
    (G 15)
    ))
)
(d G
    (q (
    (#2)
    (s #1 1)
    ))
)
(F 6)
(d H
    (q (
    (#1)
    (s #1 1)
    ))
)
(H 6)
(d I
    (q ((#1)
    ((q ((#2) (s #1 #2))) 1)
    ))
)
(I 6)