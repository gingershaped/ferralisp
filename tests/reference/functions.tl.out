#1
F
G
41
H
5
I
41